### PR TITLE
000-Setup UI repo

### PR DESCRIPTION
**Pull request by DarraghHaughey-Kainos**

### CODE REVIEW

File: service/HelloWorldService.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const axios = require("axios")
+
+module.exports.getHelloWorld = async function() {
+    try { 
+        const response = await axios.get('http://localhost:8080/api/hello-world');
```
Comment (glenn-hamilton) [inline service/HelloWorldService.ts]: Can you make `http://localhost:8080/` a configuration option. Currently if the URL is hard coded it won't work on AWS

File: controller/HelloWorldController.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+module.exports = function(app: Application) {
+
+    app.get('/hello-world', async (req: Request, res: Response) => {
+
+        let data: helloWorld[] = [];
+
+        try {
+            data = await helloWorldService.getHelloWorld();
+
+            console.log(data);
```
Comment (glenn-hamilton) [inline controller/HelloWorldController.ts]: You don't need to log the data here

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: |
+        npm install
```
Comment (glenn-hamilton) [inline .github/workflows/node.js.yml]: Could `npm ci` work here as well?

**Review by glenn-hamilton — APPROVED** [review]

File: .github/workflows/node.js.yml (hunk 3 above)
Comment (DarraghHaughey-Kainos) [reply .github/workflows/node.js.yml]: yes worked the same as install, made this change.

**Review by steveh2707 — APPROVED** [review]
looks good

**Pull request merged**